nucliadb_core/src/metrics/metric/request_time.rs

```diff
@@ -31,14 +40,15 @@ enum RequestActor {
     Relations,
 }
 
-#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
-pub struct RequestTimeKey {
-    actor: RequestActor,
-    request: String,
-}
-
 pub type RequestTimeValue = f64;
 
+type RequestCount = Family<RequestTimeKey, Counter>;
+type RequestDuration = Family<RequestTimeKey, Histogram>;
+
+const BUCKETS: [f64; 12] = [
```

Comment:
fly-by - maybe we can add a comment to explain why we have those values for the histogram

Reply:
I'm not sure about this buckets. I suppose they're a bit generic in the range of values expected. @hermeGarcia?